packages/elements/src/photon-multirx-form/photon-prescribe-workflow.tsx

```diff
@@ -39,6 +39,24 @@ import clearForm from './util/clearForm';
 import { formatPatientWeight } from './util/formatPatientWeight';
 import { checkHasPermission } from '../utils';
 
+const newMedSearchTesters = [
+  // actual testers
+  'Tim Porter', // Modern Pediatrics
+  'Rishi Khakhkhar', // Counsel
+  'Kristen Borchetta', // Blueberry Pediatrics
```

Comment:
can we comment these out before merging?

Reply:
done